roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -37,7 +39,9 @@
   template:
     src: "{{ ha_svc_template_path }}/atomic-openshift-master-{{ item }}.service.j2"
     dest: "{{ containerized_svc_dir }}/{{ openshift.common.service_type }}-master-{{ item }}.service"
-  when: openshift.master.ha is defined and openshift.master.ha | bool and openshift_master_cluster_method == "native"
+  when:
+  - openshift.master.ha is defined and openshift.master.ha | bool and openshift_master_cluster_method == "native"
+  - not openshift.common.is_master_system_container | bool
```

Comment:
Should probably break out the other and conditions to separate list items as well.
```
when:
- openshift.master.ha | default(false) | bool
- openshift_master_cluster_method == "native"
- not openshift.common.is_master_system_container | bool
```

I'm not even sure that we should even need to verify openshift_master_cluster_method, since we only support native now. @abutcher @sdodson: thoughts?